workflows-scripts/item-search-filter/InventoryItemSearch.jmx

```diff
@@ -0,0 +1,1774 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jmeterTestPlan version="1.2" properties="5.0" jmeter="5.2.1">
```

Comment:
How about using location ids which isn't connected to bugfest environment?
In order to reuse this test in another environment

Endpoint:
GET /instance-storage/instances?query=(item.effectiveLocationId=(b0237985-87e6-4f9c-b6c2-23ef426f7d03 or d695b3e4-5bbb-4ab2-b998-6e52cd395d86 or c9379617-4061-439b-80bd-117abc9f9004 or e5d578f4-17ce-4c70-b1b5-565f3605e10b or f369266a-a209-4e4a-b487-d1acf3ee6857 or bd90f838-4bc4-4ef0-963b-502210fb5976 or 1d4222af-0994-4f15-bab1-568b2f6d3f40 or c3dd9997-463b-47e3-958c-2c6fc2775f90 or 38baf4b3-4fe7-47c1-826b-5d35e7b41018)) sortby title&limit=100&offset=0

Reply:
@Svitlana-Zmiivska How do you propose acquiring locationIds from other environments that are not connected to bugfest environment? Since we are using bugfest's dataset in our perf testing environment, perhaps we should make sure that these IDs work in our perf testing environment and add a comment about them here, that they're from the PTF testing env?

Reply:
@mtraneis At least to store these locationIds in a separate file as Varun did now.
It seems to me this approach is more reusable than to store these loactionIds directly in url query

Reply:
I see what you're saying @Svitlana-Zmiivska. Good point.  I don't think the following is needed now, but maybe something that we can try in the future (because there are many checkboxes in the UI that can be selected simultaneously in many areas): we could store a long list of IDs, and have the script randomly picks a subset of those IDs on each run.